### ISSUE TITLE
Open file by Url

### ISSUE THREAD

**Issue by sadiqhabib** [sleistner/vscode-fileutils#399]
**Describe the Feature**
Open file by its url. It is more convenient to open file with url instead drag and drop.

**Comment by vrms** [comment]
If you mean open a file without having to use the mouse (same workflow as `**File Utils: Rename**` or `**File Utils: Move**`) I could not agree more.

would be really nice to have. It's actuallz surprising that it does not exist yet with the initial set of tools.

**Comment by sadiqhabib** [comment]
Yes its easy to open file like short=key -> dropdown list -> chose file -> open file

**Comment by sleistner** [comment]
Hi @sadiqhabib could you describe the expected behaviour in more detail?

**Comment by sadiqhabib** [comment]
I found `ctrl+e` shortcut key to do the same.
But my idea was you can open file using url eg
Press `ctrl+e`
navigate to file
press enter

**Comment by alexsavio** [comment]
@sadiqhabib Do you mean to be able to paste the URL of a file in the prompt, then VSCode would fetch its content for you and create a new file with said content?

**Comment by sadiqhabib** [comment]
sometimes we need to open some files in our projects, so we have 2 options
1: navigate to that file on left panel then double click to open that
2: use shortcut key ie F12 but that url must be written on some file.
But idea is open file by url which we provide on opened prompt.
